javascript/javascript1/week1/age-ify.js

```diff
@@ -0,0 +1,4 @@
+const yearOfBirth = 1986;
+let yearFuture = 2027;
+let futureAge = yearFuture - yearOfBirth;
+console.log("You will be " + futureAge + " years old in " + yearFuture + " !");
```

Comment:
Nice work. I recommend that you declare `yearFuture` & `futureAge` with const, because you will not reassign them.

Reply:
Will keep in mind onwards,thank you.